addons/sale_project/views/project_task_views.xml

```diff
@@ -67,7 +67,7 @@
             </xpath>
             <xpath expr="//field[@name='partner_phone']" position="after">
                 <field name="sale_order_id" attrs="{'invisible': True}" groups="sales_team.group_sale_salesman"/>
-                <field name="sale_line_id" string="Sales Order Item" attrs="{'invisible': [('partner_id', '=', False)]}" options='{"no_open": True}' readonly="1" context="{'create': False, 'edit': False, 'delete': False}"/>
+                <field name="sale_line_id" string="Sales Order Item" options='{"no_open": True}' readonly="1" context="{'create': False, 'edit': False, 'delete': False}"/>
```

Comment:
We also have an override in sale_timesheet module. I think we also need to update that.
https://github.com/odoo/odoo/blob/f4c054e4731270a5103ec1d5a97c2789aef977e3/addons/sale_timesheet/views/project_task_views.xml#L215

Reply:
@xavierbol Done the change.